addon-manager/tests/src/test/java/org/jboss/forge/addon/manager/AddonManagerTest.java

```diff
@@ -85,8 +86,10 @@ public static void clearRemoteRepository()
 
    @Deployment
    @AddonDeployments({
-            @AddonDeployment(name = "org.jboss.forge.addon:addon-manager"),
-            @AddonDeployment(name = "org.jboss.forge.addon:maven")
+            @AddonDeployment(name = "org.jboss.forge.addon:dependencies"),
+            @AddonDeployment(name = "org.jboss.forge.addon:maven"),
+            @AddonDeployment(name = "org.jboss.forge.addon:ui"),
```

Comment:
Is it because of your change? Could you please revert this and we might fix it in another issue?